pizone/zone.py

```diff
@@ -88,6 +88,78 @@ def airflow_min(self) -> int:
         """Min allowed airflow for the zone as a percent"""
         return self._get_zone_state('MinAir')
 
+    async def set_airflow_min(self, value: int) -> None:
+        """
+        Change the zone airflow min in 5% increments
+        Async method, returns when server has changed mode.
+        Valid values are percent in 5% increments.
+        Raises:
+            AttributeError if the set point is out of range
+        """
+        if self.type != Zone.Type.OPCL:
+            raise AttributeError(
+                'Can\'t set MinAir to \'{}\' type zone.'
+                .format(self.type))
+        if value % 5 != 0:
+            raise AttributeError(
+                'MinAir \'{}\' not rounded to nearest 5'
+                .format(value))
+        if (value < 0
+                or value > 100):
+            raise AttributeError(
+                'MinAir \'{}\' is out of range'.format(value))
+        if self._zone_data['MinAir'] == value:
+            return
+
+        async with self._controller._sending_lock:
+            if self.mode != Zone.Mode.OPCL:
```

Comment:
I don't think you actually need to check the mode here. There is no ```Mode.OPCL``` anyhow, it's ```Mode.AUTO``` and if the zone is open or closed you can still set the maximum open/close percentage.  Get rid of this check.